web/replacement_terms.json

```diff
@@ -0,0 +1,70 @@
+{
+  "definingModule": "Prelude",
```

Comment:
What about:

```
{
  "Prelude": {
    "abs": ["positive", "magnitude"],
    ...
    "translated": ["move"]
  },
  "SomeOtherFutureModule": {
    ...
  }
}
```

I don't think this is really any more work than the single-module implementation that you have now.  For now, you only have replacements for `Prelude`, but I can add more modules later.

Reply:
Sure, no problems, everything could be kept in one file.